src/helpers.ts

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
+import { BASE } from './constants';
+
+export const calculatePriceForToken = (
+  firstAmount: string,
+  secondAmount: string,
+  price: string,
+) => {
+  if (!price) return '0';
+  if (Big(firstAmount).lte(0)) return '0';
+  return new Big(firstAmount).mul(price).div(secondAmount).toFixed(5);
+};
+
+export const formatTokenAmount = (
+  value: string,
+  decimals = 18,
+  precision?: number,
+) =>
+  value &&
+  Big(value)
+    .div(Big(BASE).pow(decimals))
+    .toFixed(precision && precision);
```

Comment:
In this case `.toFixed(precision)` would have the same outcome as `.toFixed(precision && precision`.